centos-7-minimal.ks

```diff
@@ -65,10 +66,29 @@ shim
 LANG="en_US"
 echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
 
+# Make sure to properly handle the ssh keys passed to us via docker-machine
+cat > /etc/rc.d/init.d/copy-ssh << EOF
+
+# Create a docker user
+useradd -p docker docker
+
+# Read /userdata.tar with ssh keys
+dd if=/dev/sda of=/userdata.tar bs=1 count=4096 2>/dev/null
+tar xf /userdata.tar -C /home/docker/ > /var/log/userdata.log 2>&1
+rm -f '/home/docker/boot2docker, please format-me'
+#chown -R docker:staff /home/docker
+EOF
+
+chmod 755 /etc/rc.d/init.d/copy-ssh
```

Comment:
Here is the part I thought I need to make sure that the init script gets executed. I copied it from the previous kickstart file. Any idea on what the issue could be?

If we get this script executed as part of the bootstrap, we made already some progress.

Next would be to create a similar script dealing with partitioning, formatting and mounting of the disk.

@praveenkumar @LalatenduMohanty, any idea, how one would externalize this _copy-ssh_, instead of inlining it into the kickstart file?